### PR TITLE
SK-1515-TRANS-WEB-Domain-DNS-SPF 

### PR DESCRIPTION
**Pull request by BrenKt**

### CODE REVIEW

**Review by bs-ovh — APPROVED** [review]

File: pages/web_cloud/domains/dns_zone_spf/guide.es-es.md [hunk 1]
```diff
@@ -204,30 +232,35 @@ mydomain.ovh IN TXT "v=spf1 ip4:11.22.33.444 ~all"
...
->
-> Acceda a la sección `Nombres de dominio`{.action} de la columna izquierda y seleccione el dominio asociado a su plataforma Private Exchange. A continuación, abra la pestaña de la columna izquierda y haga clic en `Zona DNS`{.action}. Obtenga las direcciones IPv4 (registro `A`) y IPv6 (registro `AAAA`) correspondientes a la URL del webmail.
->
-> ![domain](/pages/assets/screens/control_panel/product-selection/web-cloud/domain-dns/dns-zone/spf_records_ip3.png){.thumbnail}
->
+/// details | ¿Cómo recuperar las direcciones IP de un servidor Private Exchange?
+
+Para obtener la dirección IP del servidor Private Exchange, haga clic en `Microsoft`{.action} y seleccione `Exchange`{.action}. y seleccione el servicio Private Exchange correspondiente.
+
+En la ficha `Información general`{.action}, haga clic en `A` y `AAAA` en la sección `Diagnóstico del servidor`. En la ventana que aparece, lea el valor
```
Comment (Y0Coss) [inline pages/web_cloud/domains/dns_zone_spf/guide.es-es.md]:
```suggestion
En la ficha `Información general`{.action}, haga clic en `A` y `AAAA` en la sección `Diagnóstico del servidor`. En la ventana que aparece, lea el valor.
```

File: pages/web_cloud/domains/dns_zone_spf/guide.es-es.md [hunk 2]
```diff
@@ -4,13 +4,32 @@ excerpt: "Descubra cómo configurar un registro SPF en un dominio para mejorar l
 updated: 2023-08-04
```
Comment (Y0Coss) [inline pages/web_cloud/domains/dns_zone_spf/guide.es-es.md]:
```suggestion
updated: 2024-09-05
```

File: pages/web_cloud/domains/dns_zone_spf/guide.es-us.md [hunk 3]
```diff
@@ -4,13 +4,32 @@ excerpt: "Descubra cómo configurar un registro SPF en un dominio para mejorar l
 updated: 2023-08-04
```
Comment (Y0Coss) [inline pages/web_cloud/domains/dns_zone_spf/guide.es-us.md]:
```suggestion
updated: 2024-09-05
```

File: pages/web_cloud/domains/dns_zone_spf/guide.es-us.md [hunk 4]
```diff
@@ -202,25 +230,26 @@ mydomain.ovh IN TXT "v=spf1 ip4:11.22.33.444 ~all"
...
->
-> Acceda a la sección `Nombres de dominio`{.action} de la columna izquierda y seleccione el dominio asociado a su plataforma Private Exchange. A continuación, abra la pestaña de la columna izquierda y haga clic en `Zona DNS`{.action}. Obtenga las direcciones IPv4 (registro `A`) y IPv6 (registro `AAAA`) correspondientes a la URL del webmail.
->
-> ![domain](/pages/assets/screens/control_panel/product-selection/web-cloud/domain-dns/dns-zone/spf_records_ip3.png){.thumbnail}
->
+/// details | ¿Cómo recuperar las direcciones IP de un servidor Private Exchange?
+
+Para obtener la dirección IP del servidor Private Exchange, haga clic en `Microsoft`{.action} y seleccione `Exchange`{.action}. y seleccione el servicio Private Exchange correspondiente.
+
+En la ficha `Información general`{.action}, haga clic en `A` y `AAAA` en la sección `Diagnóstico del servidor`. En la ventana que aparece, lea el valor
```
Comment (Y0Coss) [inline pages/web_cloud/domains/dns_zone_spf/guide.es-us.md]:
```suggestion
En la ficha `Información general`{.action}, haga clic en `A` y `AAAA` en la sección `Diagnóstico del servidor`. En la ventana que aparece, lea el valor.
```

File: pages/web_cloud/domains/dns_zone_spf/guide.fr-ca.md [hunk 5]
```diff
@@ -4,9 +4,28 @@ excerpt: Découvrez comment configurer un enregistrement SPF sur votre nom de do
 updated: 2023-08-04
```
Comment (Y0Coss) [inline pages/web_cloud/domains/dns_zone_spf/guide.fr-ca.md]:
```suggestion
updated: 2024-09-05
```

File: pages/web_cloud/domains/dns_zone_spf/guide.it-it.md [hunk 6]
```diff
@@ -4,13 +4,32 @@ excerpt: "Questa guida ti mostra come configurare un record SPF sul tuo dominio
 updated: 2023-08-04
```
Comment (Y0Coss) [inline pages/web_cloud/domains/dns_zone_spf/guide.it-it.md]:
```suggestion
updated: 2024-09-05
```

File: pages/web_cloud/domains/dns_zone_spf/guide.pl-pl.md [hunk 7]
```diff
@@ -4,15 +4,34 @@ excerpt: "Dowiedz się, jak skonfigurować rekord SPF dla Twojej domeny, aby zwi
 updated: 2023-08-04
```
Comment (Y0Coss) [inline pages/web_cloud/domains/dns_zone_spf/guide.pl-pl.md]:
```suggestion
updated: 2024-09-05
```

File: pages/web_cloud/domains/dns_zone_spf/guide.pl-pl.md [hunk 8]
```diff
@@ -78,23 +108,23 @@ W tabeli, aby odnaleźć wiersz odpowiadający SPF OVHcloud, można użyć filtr
 > SPF zawsze składa się z następującej formy: "v=spf1 `sources` `qualifieur`". Na przykład, SPF OVHcloud to: "v=spf1 include:mx.ovh.com ~all".
```
Comment (Y0Coss) [inline pages/web_cloud/domains/dns_zone_spf/guide.pl-pl.md]: `qualifieur` may need to be translated in all languages

File: pages/web_cloud/domains/dns_zone_spf/guide.pt-pt.md [hunk 9]
```diff
@@ -4,13 +4,32 @@ excerpt: "Saiba como configurar um registo SPF no seu domínio para melhorar a s
 updated: 2023-08-04
```
Comment (Y0Coss) [inline pages/web_cloud/domains/dns_zone_spf/guide.pt-pt.md]:
```suggestion
updated: 2024-09-05
```

**Review by Y0Coss — APPROVED** [review]

**Pull request merged**